### PR TITLE
swaps out soulstone tendril loot for a new one called seismic right arm

### PR DESCRIPTION
**Pull request by LazennG**
# Document the changes in your pull request
takes soulshard out of the tendril loot pool and throws it in the dumpster while introducing a new item called seismic arm.
Seismic right arm is a mechanical right arm that gives the user 4 abilities
Lariat:
dreamseeker_72hlQ1qnKP
Move ahead 5 spaces and deal 50 damage to whatever animal you hit and 10 to people, has a 7 second cooldown.

Mop the floor:
dreamseeker_dvtCdklNFp

Similar to lariat, this move sends the user ahead 5 spaces, but if they connect with an animal they start dealing 20 damage per tile travelled together and the user gets the blood-drunk miner trophy effect to help negate damage because of guaranteed prolonged exposure and because fauna can pile up if they're lined up closely. Dead fauna get gibbed. People hit by this take 4 damage per tile and the user doesn't get the blood drunk effect. The cooldown for this move is also 7 seconds

Suplex:
dreamseeker_YsTq9JNIgk

This move targets a single target right in front of the user and puts them behind the user. If the target is an animal they'll take 20 damage and if the target is a person they'll take 6. If an animal is right behind the user when the move is used successfully, they'll take 20 damage, and if a person is there they'll take 10. This move also gibs dead animals and has a 1 second cooldown.

Right Hook:
dreamseeker_2CdJxqZevo
The user channels the ability for 2 seconds to get an inhand item called 'overcharged emitter' which lasts 5 seconds. On hitting a target, the user's right arm is disabled for 15 seconds and none of the abilities of the arm can be used until it's functional. Targets hit will be knocked back 8 spaces. Animals take 100 damage on impact and people take 15. Anybody caught in the flight path of the initial target will also be knocked backwards and take 10 damage. 

'Suplex, 'mop the floor,' and 'right hook' will dig through rocks if the target is sent into them and deal an additional 5 damage per rock.

Like most lavaland bodyparts, the arm needs to be attached surgically unless the user is an IPC, and because it's a mechanical limb it shares the same weaknesses and needs the same upkeep.

# Spriting
Sprite for the arm as an item:
r_arm_item_seismic
Sprites for the arm when attached to a moja
image
Action button sprites by @thegoldencat413
image
# Wiki Documentation

https://wiki.yogstation.net/wiki/Shaft_Miner#Necropolis_Chest_Loot
under the loot section, replace the soulstone with the seismic right arm. 
For the description, something like: A mechanical right arm that gives the user 4 abilities when surgically attached
-Lariat: The user dashes 5 spaces ahead and hits fauna for 50 damage while hitting people for 10
-Mop the floor: The user dashes 5 spaces ahead and pushes whoever it hits, dealing 20 damage per traveled tile to animals and 4 to people. Dead animals get gibbed by this move
-Suplex: The user slams the target behind them, dealing 20 damage to fauna and 6 to people. If there's somebody in the landing zone, they'll take 20 damage if they're an animal and 10 if they're a person. Dead animals get gibbed by this move 
-Right Hook: The user channels the ability for 2 seconds before getting an item that lasts 5 seconds in their right hand. Hitting somebody with this move sends them flying 8 spaces. Animals take 100 damage on hit and people take 15. If anybody is in the flight path, they get taken along with the original target and take 10 damage.

Mop the floor, suplex, and right hook deal an additional 5 damage for every rock the target hits.

# Changelog

:cl: @LazennG   @thegoldencat413 @ToasterBiome @Chubbygummibear 
rscadd: Added seismic right arm 
rscdel: Removes soulstone from tendril loot pool 
imageadd: added sprites for the arm and its abilities
/:cl:
Big thanks to @thegoldencat413 for extensive testing and feedback, along with the actionbutton sprites, and to baiomu and chubby for code help. Also thanks to @cuackles for spriting the arm's item sprite into something cooler than the original blue recolor of the default borg arm

### CODE REVIEW

**Comment by thegoldencat413** [comment]
holy shit... this is the best thing ive ever seen... this cured my cancer... speed merge this now ............

**Comment by Mqiib** [comment]
This is super cool for two reasons.
1: It's super cool mining loot that looks like it'll be pretty useful against fauna, or at least fun to play with
2: I can steal code from it for my own nefarious plans

Bonus part 3: It's specifically set to not do a ton of damage to people on-station while still having interesting effects. That being said it might be relevant to make sure it respects armor

**Comment by Moltijoe** [comment]
Very cool, i appreciate anything that makes being an ipc/preternis miner worthwhile considering the lack of medipens

**Comment by LazennG** [comment]
dreamseeker_cyoeHE7r1P
Suplex tweaked a bit so it turns targets on their head now, borg damage for lariat, mop, suplex, and righthook is now 12,5,8, and 18 respectively.
Also, now if an animal caught in righthook is sent flying into a solid object and is dead, they explode into gibs.
Damage for people caught in right hook has been tweaked as well. If the target flies into a person, the person takes 10 damage. Borgs would take 11 damage, and animals take 30.
dreamseeker_7euEpE338m
dreamseeker_c3kSCS7DIX
dreamseeker_GZxBPSPzNx

**Comment by ShoziOffical** [comment]
NOT AXOLOTLS  you monster

**Review by JamieD1 — APPROVED** [review]

**Review by JamieD1 — DISMISSED** [review]
Hope all this only works on simple mobs, if it doesnt no way is this being added. Cant be instagibbing humans.

**Comment by LazennG** [comment]
> Hope all this only works on simple mobs, if it doesnt no way is this being added. Cant be instagibbing humans.

None of the moves gib carbons or borgs and the moves deal largely reduced damage to both like stated before dw

**Comment by LazennG** [comment]
item sprite for the arm has been updated with one by @cuackles :
THEHANDO
image
image

**Comment by LazennG** [comment]
> item sprite for the arm has been updated with one by @cuackles : THEHANDO image image

belaying the change for now since it's not vital and i dont want to keep delaying, so the item sprite is back to the tiny blue arm in the original post

**Comment by KillerOrcaCora** [comment]
@JamieD1 can use another review or testmerge, PR never did what was asked to be changed in request

File: code/modules/mining/lavaland/seismicarm.dm [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "flagellation"
+	item_state = "hivehand"
+	color = "#04b8ff"
+	item_flags = DROPDEL
+	w_class = 5
+
+/obj/item/overcharged_emitter/afterattack(mob/living/L, mob/living/user, proximity)
+	. = ..()
+	var/flightdist = 8
```
Comment (Bibby0110) [inline code/modules/mining/lavaland/seismicarm.dm]: Should probably be a var on the item

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
An attempt should probably be made to reduce the amount of copy-paste by making the abilities share a parent

**Review by cuackles — APPROVED** [review]
Art

**Comment by ReddicusDragon** [comment]
I worry about this potentially being powergamey being an antag miner, other than that it's really freaking cool

**Comment by thegoldencat413** [comment]
> I worry about this potentially being powergamey being an antag miner, other than that it's really freaking cool

disagree because the highest damage attack is 10 brute, which is less than a toolbox, and that puts all the abilities on a long cooldown. The most spammable attack has a 1 second cooldown and does 6 damage, which is less than a punch deals on average, on a longer cooldown.

EMP weakness is also a massive liability

im super biased though because i think this item is cool as hell

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**